manager/src/main/java/com/redhat/service/smartevents/manager/dns/openshift/DnsConfigOpenshiftProviderImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.redhat.service.smartevents.manager.dns.openshift;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.route53.AmazonRoute53Async;
+import com.amazonaws.services.route53.AmazonRoute53AsyncClientBuilder;
+
+import io.quarkus.arc.properties.IfBuildProperty;
+
+@ApplicationScoped
+@IfBuildProperty(name = "event-bridge.k8s.orchestrator", stringValue = "openshift", enableIfMissing = true)
```

Comment:
@r00ta Not sure if the `enableIfMissing` parameter has any effect here as we also provide a default in `application.properties` and at the minute that default is `minikube` (I don't think it should be `minikube` by default).